website/static/_redirects

```diff
@@ -0,0 +1,3 @@
+# https://docs.netlify.com/routing/redirects/
+
+/contributing.md /about/contributing.md 301
```

Comment:
```suggestion
/contributing.md /about/contributing.md 302
```

I have horror memories of 301s being literally impossible to evict from browser caches, and having been unable to make changes to a site structure in the past because 301s are for reals permanent :(